sdks/python/README.md

```diff
@@ -24,66 +51,68 @@ from pprint import pprint
 import requests
 import yaml
 
-import openapi_client
-from openapi_client.api import workflow_service_api
-from openapi_client.model.io_argoproj_workflow_v1alpha1_workflow_create_request import \
+import argo_workflows
+from argo_workflows.api import workflow_service_api
+from argo_workflows.model.io_argoproj_workflow_v1alpha1_workflow_create_request import
     IoArgoprojWorkflowV1alpha1WorkflowCreateRequest
 
-configuration = openapi_client.Configuration(host="https://127.0.0.1:2746")
+configuration = argo_workflows.Configuration(host="https://127.0.0.1:2746")
 configuration.verify_ssl = False
 
 resp = requests.get('https://raw.githubusercontent.com/argoproj/argo-workflows/master/examples/hello-world.yaml')
 manifest = yaml.safe_load(resp.text)
 manifest['spec']['serviceAccountName'] = 'argo'
 
-api_client = openapi_client.ApiClient(configuration)
+api_client = argo_workflows.ApiClient(configuration)
 api_instance = workflow_service_api.WorkflowServiceApi(api_client)
-api_response = api_instance.workflow_service_create_workflow(
-	namespace='argo',
-	body=IoArgoprojWorkflowV1alpha1WorkflowCreateRequest(
-		workflow=manifest, _check_type=False))
+api_response = api_instance.create_workflow(
+    namespace='argo',
+    body=IoArgoprojWorkflowV1alpha1WorkflowCreateRequest(
+        workflow=manifest, _check_type=False))
 pprint(api_response)
 ```
 
 Note that `_check_type=False` is required here to avoid type checks against `manifest` which is a Python dictionary.
 
-Alternative, you can submit a workflow with an instance of `IoArgoprojWorkflowV1alpha1Workflow` constructed via the SDK like the following:
+Alternative, you can submit a workflow with an instance of `IoArgoprojWorkflowV1alpha1Workflow` constructed via the SDK
+like the following:
 
 ```python
 from pprint import pprint
 
 import openapi_client
 from openapi_client.api import workflow_service_api
 from openapi_client.model.container import Container
-from openapi_client.model.io_argoproj_workflow_v1alpha1_template import \
+from openapi_client.model.io_argoproj_workflow_v1alpha1_template import
     IoArgoprojWorkflowV1alpha1Template
 from openapi_client.model.io_argoproj_workflow_v1alpha1_workflow import (
-    IoArgoprojWorkflowV1alpha1Template, IoArgoprojWorkflowV1alpha1Workflow)
-from openapi_client.model.io_argoproj_workflow_v1alpha1_workflow_create_request import \
+    IoArgoprojWorkflowV1alpha1Template, IoArgoprojWorkflowV1alpha1Workflow
+)
+from openapi_client.model.io_argoproj_workflow_v1alpha1_workflow_create_request import
     IoArgoprojWorkflowV1alpha1WorkflowCreateRequest
-from openapi_client.model.io_argoproj_workflow_v1alpha1_workflow_spec import \
+from openapi_client.model.io_argoproj_workflow_v1alpha1_workflow_spec import
```

Comment:
The import needs to be updated to use `argo-workflows`. Also could you confirm the examples are still working as expected (besides the known issue I mentioned previously)?